gym_solo/core/obs.py

```diff
@@ -13,6 +14,10 @@
 
 
 class Observation(ABC):
+  """An observation for a body in the pybullet simulation.
+  """
+  _client: bullet_client.BulletClient = None
```

Comment:
A little context on why is this change needed?

Reply:
I understood from how you are using it but it would be nice to have a one-liner comment here

Reply:
done